src/test/groovy/edu/ie3/datamodel/io/naming/EntityPersistenceNamingStrategyTest1.groovy

```diff
@@ -0,0 +1,529 @@
+import edu.ie3.datamodel.models.input.connector.type.LineTypeInput
+import edu.ie3.datamodel.models.input.connector.type.Transformer2WTypeInput
+import edu.ie3.datamodel.models.input.connector.type.Transformer3WTypeInput
+import edu.ie3.datamodel.models.input.graphics.LineGraphicInput
+import edu.ie3.datamodel.models.input.graphics.NodeGraphicInput
+import edu.ie3.datamodel.models.input.system.BmInput
+import edu.ie3.datamodel.models.input.system.ChpInput
+import edu.ie3.datamodel.models.input.system.EvInput
+import edu.ie3.datamodel.models.input.system.EvcsInput
+import edu.ie3.datamodel.models.input.system.FixedFeedInInput
+import edu.ie3.datamodel.models.input.system.HpInput
+import edu.ie3.datamodel.models.input.system.LoadInput
+import edu.ie3.datamodel.models.input.system.PvInput
+import edu.ie3.datamodel.models.input.system.StorageInput
+import edu.ie3.datamodel.models.input.system.WecInput
+import edu.ie3.datamodel.models.input.system.characteristic.EvCharacteristicInput
+import edu.ie3.datamodel.models.input.system.characteristic.WecCharacteristicInput
+import edu.ie3.datamodel.models.input.system.type.BmTypeInput
+import edu.ie3.datamodel.models.input.system.type.ChpTypeInput
+import edu.ie3.datamodel.models.input.system.type.EvTypeInput
+import edu.ie3.datamodel.models.input.system.type.HpTypeInput
+import edu.ie3.datamodel.models.input.system.type.StorageTypeInput
+import edu.ie3.datamodel.models.input.system.type.WecTypeInput
+import edu.ie3.datamodel.models.input.thermal.CylindricalStorageInput
+import edu.ie3.datamodel.models.input.thermal.ThermalHouseInput
+import edu.ie3.datamodel.models.result.NodeResult
+import edu.ie3.datamodel.models.result.connector.LineResult
+import edu.ie3.datamodel.models.result.connector.SwitchResult
+import edu.ie3.datamodel.models.result.connector.Transformer2WResult
+import edu.ie3.datamodel.models.result.connector.Transformer3WResult
+import edu.ie3.datamodel.models.result.system.BmResult
+import edu.ie3.datamodel.models.result.system.ChpResult
+import edu.ie3.datamodel.models.result.system.EvResult
+import edu.ie3.datamodel.models.result.system.EvcsResult
+import edu.ie3.datamodel.models.result.system.FixedFeedInResult
+import edu.ie3.datamodel.models.result.system.LoadResult
+import edu.ie3.datamodel.models.result.system.PvResult
+import edu.ie3.datamodel.models.result.system.StorageResult
+import edu.ie3.datamodel.models.result.system.WecResult
+import edu.ie3.datamodel.models.result.thermal.CylindricalStorageResult
+import edu.ie3.datamodel.models.result.thermal.ThermalHouseResult
+import edu.ie3.datamodel.models.timeseries.IntValue
+import edu.ie3.datamodel.models.timeseries.individual.IndividualTimeSeries
+import edu.ie3.datamodel.models.timeseries.individual.TimeBasedValue
+import spock.lang.Specification
+
+import java.nio.file.Paths
+import java.time.ZonedDateTime
+import java.util.regex.Pattern
+
+class EntityPersistenceNamingStrategyTest1 extends Specification {
```

Comment:
Why did you split up the test tasks? If it still is sensible, please give it a reasonable naming.

Reply:
Actually, the number of functions in `EntityPersistenceNamingStrategyTest.groovy` was more than 30 due to which Codacy raised an issue. I decided to split the test cases into two files to fix the issue but I could not come up with some way in which I can decide the test case files to have a meaningful name. I will merge them both together again.